src/ontology/mondo-edit.obo

```diff
@@ -171331,17 +171338,10 @@ property_value: confidence "1.013902333975493" xsd:double
 
 [Term]
 id: MONDO:0009148
-name: Rosselli-Gulienetti syndrome
-def: "A rare congenital ectodermal dysplasia syndrome with a range of signs and symptoms including cleft lip or palate, mental retardation and various forms of ectodermal dysplasia. Additional symptoms may include fused eyelids, absent nails, delayed bone growth and dry skin. It is believed that this syndrome follows an autosomal dominant pattern of inheritance with incomplete penetrance, and caused by a mutation affecting the TP63 gene" [https://en.wikipedia.org/wiki/Rosselli%E2%80%93Gulienetti_syndrome]
+name: obsolete Rosselli-Gulienetti syndrome
 comment: obsolete in Orphanet. Wikipedia suggests a merge: it has been suggested that this syndrome, AEC syndrome and Rapp–Hodgkin syndrome may be variations of the same disease
```

Comment:
I may have written the original comment. In cases like this where we have disagreement I would state this in terms of the clinical/medical reasons for merging, I would not note wikipedita